pkgs/development/python-modules/pytest-grpc/default.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, grpcio
+, pytest
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-grpc";
+  version = "0.8.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-C9JoP/00GZRE1wfAqwGXCyLgr7umyx3bbVeMhev+Cb0=";
+  };
+
+  propagatedBuildInputs = [
+    grpcio
+    pytest
+  ];
```

```suggestion

  buildInputs = [
    pytest
  ];

  propagatedBuildInputs = [
    grpcio
  ];
```
special handling for pytest plugins otherwise we cannot overwrite the pytest version easily